x/signal/keeper_test.go

```diff
@@ -122,11 +121,13 @@ func TestTallyingLogic(t *testing.T) {
 		Version:          0,
 	})
 	require.Error(t, err)
+	require.ErrorIs(t, err, types.ErrInvalidVersion)
```

Comment:
[nit] Would be helpful to add a one liner for each testcase explaining why an error is required or not required.